prisma/schema/application.prisma

```diff
@@ -33,8 +33,8 @@ enum AlevelQualification {
 
 model Application {
   id                       Int                  @id @default(autoincrement())
-  applicant                Applicant            @relation(fields: [applicantId], references: [id])
-  applicantId              Int
+  applicant                Applicant            @relation(fields: [cid], references: [cid])
```

Comment:
:OOOO We are linking by CID and not by ID? I think there's a bigger choice to make here: do we use CID as primary key for Applicant or do we not?

Reply:
Having _applicantId_ as a foreign key before was redundant given that applications should have CIDs directly on them and there are two unique fields in`applicant.prisma`,  _id_ and _cid_. Given how the College system and the specification given, I think we could move to using CID as the primary key.

Reply:
That's ok. Then I would issue a dedicated PR with this migration (using CID as primary key for Applicant and adjusting all the relationships accordingly). Once that is merged in, we rebase this current branch onto main: that should help shrinking it a little, too :)

Reply:
resolved in 81d27c99fc655a1dfdecdd4d4642e9bb3f2d8418